public/views/address.html

```diff
@@ -16,16 +16,29 @@
       </a>
     </div>
   </div>
-  <h1>Address<small data-ng-if="address.balance >= 0">&nbsp;{{address.balance | lisk}} LSK</small></h1>
-  <div class="text-muted" data-ng-if="!(address.balance >= 0)">
-    <span>Loading Address <i class="fa fa-spinner fa-spin"></i></span>
+  <h2>Address:&nbsp;<small>{{address.address}}</small></h2>
+        <div class="text-muted" data-ng-if="!(address.balance >= 0)">
```

Comment:
Because you changed `address.balance` to `address.address` that check is no longer viable.